base_app/base.py

```diff
@@ -66,5 +66,27 @@ def get_photos():
 
     return result
 
+@app.route('/api/v1/random', methods=['GET'])
+def get_random():
+    YELP_API_KEY = os.getenv('YELP_API_KEY')
+    headers = {'authorization': 'Bearer ' + YELP_API_KEY}
+    lat = request.args['latitude']
+    long = request.args['longitude']
+    business_type = 'food'
+    open = True
+    params = {'latitude': f'{lat}', 'longitude': f'{long}', 'term': f'{business_type}', 'open_now': f'{open}'}
```

Comment:
I like the params here for the random - it's going to pull food related businesses but also broad enough. I know you mentioned adding the open now param to the other api call as well. Another thought - if there are a ton of food related businesses, we might want to add a limit to the params and return maybe 20?

Reply:
By default, the returned results are limited to 20. Another option for `business_type` would be `restaurants`, if we wanted to be more explicit. Yelp's search algorithm does sort by best match by default.